recipes/ydata-synthetic/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ydata-synthetic" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ydata-synthetic-{{ version }}.tar.gz
+  sha256: 90db2ff3eb6d9279a1e719db40574a8ff36f1564131393a012dbc5a77e7e7ba2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.9
+    - pip
+  run:
+    - python >=3.7,<3.9
+    - pandas >=1.3.5
+    - numpy >=1.22.0
+    - scikit-learn >=1.0.0
+    - matplotlib-base >=3.5.0
+    - tensorflow-cpu >=2.8.0
+    - easydict >=1.9
+    - pmlb >=1.0
+    - tqdm
+    - typeguard >=2.13.0
+    - pytest >=6.2.0
+    - requests >=2.24.0
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
Pip Check will be very restrictive and fail, as the requirements.txt is too locked down
requests>=2.24.0, <2.29
pandas==1.4.*
numpy==1.23.*
scikit-learn==1.1.*
matplotlib==3.5.*
tensorflow==2.9.0
easydict==1.9
pmlb==1.0.*
tqdm<5.0
typeguard==2.13.*
pytest==6.2.*

Reply:
Please then raise an issue about this upstream and mention it here.